packages/http/src/csrfHandling.js

```diff
@@ -57,12 +53,12 @@ function parseCookie(cookie) {
  * @param {Object.CSRFTokenCookieKey} CSRFTokenCookieKey - default `csrfToken`
  * @param {Map.<string, string>} cookieValues
  */
-const findCSRFToken = curry(({ CSRFTokenCookieKey = 'csrfToken' }, cookieValues) => {
+const findCSRFToken = ({ CSRFTokenCookieKey = 'csrfToken' }) => cookieValues => {
```

Comment:
curry do not mean that.
You may break users of http here

Reply:
do you have an example of what could go wrong? This function is not exported and I made sur the only usage is OK